<?xml version="1.0" encoding="utf-8"?>
<launch>

   <node name="rviz" pkg="rviz" type="rviz" output="screen"
    args="-d $(find gcopter_ml)/config/global_planning.rviz"> 
  </node>
  
  <node name="pos_vel_plot" pkg="rqt_plot" type="rqt_plot" 
    args="/visualizer/speed /visualizer/total_thrust /visualizer/tilt_angle /visualizer/body_rate">
  </node>

  <!-- <node pkg="mockamap" type="mockamap_node" name="mockamap_node" output="screen">
    <param name="seed" type="int" value="1024"/>
    <param name="update_freq" type="double" value="1.0"/>
    <param name="resolution" type="double" value="0.25"/>
    <param name="x_length" type="int" value="40"/>
    <param name="y_length" type="int" value="40"/>
    <param name="z_length" type="int" value="5"/>
    <param name="type" type="int" value="1"/>
    <param name="complexity"    type="double" value="0.02"/>
    <param name="fill"          type="double" value="0.25"/>
    <param name="fractal"       type="int"    value="1"/>
    <param name="attenuation"   type="double" value="0.1"/>
    <remap from="/mock_map" to="/voxel_map"/>
  </node> -->
  <!-- <node pkg="mockamap" type="mockamap_node" name="mockamap_node" output="screen">
      <rosparam file="$(find mockamap)/config/mockamap.yaml"/>
      <remap from="/mock_map" to="/voxel_map"/>
  </node> -->

  <node pkg="nodelet" type="nodelet" name="manager"  args="manager" output="screen">
    <param name="num_worker_threads" value="16"/>
  </node>

  <node pkg="nodelet" type="nodelet" name="global_planning" args="load gcopter/GlobalPlanner manager" output="screen">
    <rosparam file="$(find gcopter_ml)/config/global_planning.yaml" command="load" />
  </node>    

</launch>
